### ISSUE TITLE
$viewContentLoading is not being fired

### ISSUE THREAD

**Issue by abuteler** [angular-ui/ui-router#3282]
Hi there.
I have this code:
```
angular.module('app')

  .run(($rootScope) => {

    $rootScope.$on('$viewContentLoading', (event, viewConfig) => {
      console.log('loading the view');
    });

    $rootScope.$on('$viewContentLoaded', (event) => {
      console.log('view has loaded');
    });

  });
```
When running the app, while I can see the logs saying that the view has loaded each time I navigate to a new section/state, I somehow never get the loading event logged.
Searched the web and this repo's past issues for other occurrences of this problem and found nothing.
Am I missing something? Do I need to fire the event manually from the routes' configs?
Thanks!

**Comment by christopherthielen** [comment]
what version of ui-router

**Comment by abuteler** [comment]
1.0.0-beta.1

**Comment by christopherthielen** [comment]
Should have been fixed by c346a284d2d7c9e9273f6d0ea2073ef1f5c131fd in 1.0.0-rc.1
